install.sh

```diff
@@ -84,6 +101,19 @@ ask () {
     done
 }
 
+try-restart-xochitl () {
```

Comment:
change to `try_restart_xochitl`

Reply:
Ah, because this is running with sh instead of bash, where this is a valid name. If you want to recommend changes in the future, you can use the ![image](https://github.com/mb1986/rm-hacks/assets/1176979/4dc7b067-1fae-44f9-b3b2-d61e0694d125) button to create a suggested change that can be applied easily.